### PR TITLE
chore: add contract date to test index

### PR DESCRIPTION
**Pull request by ada-krupa**

### CODE REVIEW

File: src/test/module/home.document.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { IsString, IsEnum, IsNumber, IsBoolean, IsOptional } from 'class-validator'
+import { IsString, IsEnum, IsNumber, IsBoolean, IsOptional, IsDate } from 'class-validator'
```
Comment (przemyslawwalczak) [inline src/test/module/home.document.ts:1]: let's remove it if not needed

File: src/test/scripts/generate-random-data.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ const data = new Array(DOCUMENTS_COUNT).fill(null).map((): HomeDocument => {
     const hasProperty = faker.number.int({ min: 0, max: 1 })
     const hasPropertyAreaSquared = faker.number.int({ min: 0, max: 1 })
     const areaSquared = faker.number.int({ min: 1, max: 1_000_000 })
+    const contractDate = faker.date.between({ from: '2023-01-01', to: '2023-12-31' }).toISOString()
```
Comment (przemyslawwalczak) [inline src/test/scripts/generate-random-data.ts:20]: does it have to be converted to iso string? 🤔

File: src/test/scripts/generate-random-data.ts [hunk 3]
```diff
@@ -32,7 +33,9 @@ const data = new Array(DOCUMENTS_COUNT).fill(null).map((): HomeDocument => {
         // eslint-disable-next-line @typescript-eslint/ban-types
         propertyAreaSquared: hasProperty && hasPropertyAreaSquared ? areaSquared : (null as unknown as undefined),
         // eslint-disable-next-line @typescript-eslint/ban-types
-        propertyAreaSquaredAsString: hasProperty && hasPropertyAreaSquared ? areaSquared.toString() : (null as unknown as undefined)
+        propertyAreaSquaredAsString: hasProperty && hasPropertyAreaSquared ? areaSquared.toString() : (null as unknown as undefined),
```
Comment (przemyslawwalczak) [inline src/test/scripts/generate-random-data.ts:36]: why do we convert `areaSquared` to string?

Comment (przemyslawwalczak) [reply src/test/scripts/generate-random-data.ts:36]: oh okay, it is being added `...AsString` field - noted 🙏

Comment (ada-krupa) [reply src/test/scripts/generate-random-data.ts:36]: yes, I guess that's what you wanted to do 😃

File: src/test/scripts/generate-random-data.ts (hunk 2 above)
Comment (ada-krupa) [reply src/test/scripts/generate-random-data.ts:20]: yes, I want it to be typed as date in elasticsearch, not as text

**Review by przemyslawwalczak — APPROVED** [review]

**Pull request merged**